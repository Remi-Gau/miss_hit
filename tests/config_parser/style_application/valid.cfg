enable_rule: "file_length"
suppress_rule: "line_length"
