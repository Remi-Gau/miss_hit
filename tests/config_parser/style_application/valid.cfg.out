MISS_HIT configuration file
  Style application for file_length
    Enabled: True
  Style application for line_length
    Enabled: False
MISS_HIT Debug Summary: 1 file(s) analysed, everything seems fine
